include/rocksdb/advanced_options.h

```diff
@@ -773,6 +774,30 @@ struct AdvancedColumnFamilyOptions {
   // Dynamically changeable through the SetOptions() API
   CompressionType blob_compression_type = kNoCompression;
 
+  // UNDER CONSTRUCTION -- DO NOT USE
+  // Enables garbage collection of blobs. Blob GC is performed as part of
+  // compaction. Valid blobs residing in blob files older than a cutoff get
+  // relocated to new files as they are encountered during compaction, which
+  // makes it possible to clean up blob files once they contain nothing but
+  // obsolete/garbage blobs. See also blob_garbage_collection_cutoff below.
+  //
+  // Default: false
+  //
+  // Dynamically changeable through the SetOptions() API
+  bool enable_blob_garbage_collection = false;
+
+  // UNDER CONSTRUCTION -- DO NOT USE
+  // The cutoff in terms of blob file age for garbage collection. Blobs in
+  // the oldest N blob files will be relocated when encountered during
+  // compaction, where N = garbage_collection_cutoff * number_of_blob_files.
+  // Note that enable_blob_garbage_collection has to be set in order for this
+  // option to have any effect.
+  //
+  // Default: 0.25
+  //
+  // Dynamically changeable through the SetOptions() API
+  double blob_garbage_collection_cutoff = 0.25;
```

Comment:
From the name, what `cutoff` actually refers to is kind-of unclear. Is it possible to give it a more explicit name?

Reply:
Same question, what is the expected behavior is application adjusts this cutoff in the middle of a compaction?

Reply:
> From the name, what `cutoff` actually refers to is kind-of unclear. Is it possible to give it a more explicit name?

How about putting "age" in there, i.e. `blob_garbage_collection_age_cutoff` ?

Reply:
I can't think of another, and `age` sounds good to me.